operator/main.go

```diff
@@ -254,7 +254,10 @@ func runOperator() {
 				if identity == operatorID {
 					log.Info("Leading the operator HA deployment")
 				} else {
-					log.WithField("operator-id", operatorID).Infof("Operator with ID %q elected as new leader", identity)
+					log.WithFields(logrus.Fields{
+						"newLeader":  identity,
```

Comment:
Too late, but isn't a consensus to declare logfield consts in `pkg/logging/logfields`, and then use them as keys? /cc @aanm

Reply:
@brb I don't see how we can enforce it if we don't have a automatic way to verify it.